docs/source/tutorial_tipsntricks.rst

```diff
@@ -44,6 +44,39 @@ That is, by adding a colon ":" followed by the flag name, an equals sign, and th
 You can add multiple flags, all separated by ":".
 
 
+Self-Chats
+##########
+
+Sometimes it is useful to generate models talking to themselves. You can do this with:
```

Comment:
could you maybe elaborate a bit on why you would specify something like `convai2:selfchat`? and perhaps include other tasks for which this is a thing?

Reply:
I have written that below:

The task set by '-t' (in the above case "convai2:selfchat") links to a parlAI world that handles the particular nature of interactions, see e.g. `here <https://github.com/facebookresearch/ParlAI/blob/master/parlai/tasks/convai2/worlds.py#L98>`_ 
or `here <https://github.com/facebookresearch/ParlAI/blob/master/parlai/tasks/wizard_of_wikipedia/worlds.py#L106>`_.

Reply:
not sure how i missed that, sorry

Reply:
well it was kind of there, but i updated it also!